asset/asset.go

```diff
@@ -459,6 +463,21 @@ type GroupKey struct {
 	Sig schnorr.Signature
 }
 
+// GroupKeyReveal is a type for representing the data used to derive the tweaked
+// key used to identify an asset group. The final tweaked key is the result of:
+// TapTweak(internalTweakedKey, tapscriptRoot)
+type GroupKeyReveal struct {
+	// InternalTweakedKey is the tweaked public key that is a result of:
+	// groupInternalKey + sha256(groupInternalKey || assetID) * G.
+	InternalTweakedKey SchnorrKey
```

Comment:
This should be the non-tweaked key? (called `asset_raw_group_key`) in the BIP. Otherwise one cannot it has been tweaked with the `asset_id`

Reply:
Ah true, I think I was mixing this up with `GroupKey.GroupPubKey`, which would now be the doubly-tweaked key.